scripts/ci-ai-model-serving/tests/04-verify-device-plugin.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+oc rollout status -n nvidia-device-plugin daemonset/nvidia-device-plugin-daemonset
+
+capacity=$(oc get node -o json | jq -r '.items[0].status.capacity')
+gpus=$(echo "${capacity}" | jq -r '."nvidia.com/gpu"')
+
+if [[ "${gpus}" != "1" ]]; then
```

Comment:
Should we check that it's not empty? I mean there may be more than one GPUs on the node.